app/assets/stylesheets/content/_news.sass

```diff
@@ -33,5 +33,18 @@
   @include text-shortener
   margin-bottom: 5px
 
-.summary
+.news-summary
```

Comment:
You changed the class name here, however the class `news-summary` is never used. So either you look for all places where `summary` is used and replace it with `news-summary` or you undo this particular change. I would recommend the second option, as there are a lot of potential places where the class is used, and it is hard to look for a class named "summary".